Formula/convox@2.rb

```diff
@@ -0,0 +1,39 @@
+class Convox < Formula
```

Comment:
This won't work. In addition to this, will v2 be supported by upstream for a while next to v3?